TestSuite/pack.py

```diff
@@ -216,14 +216,17 @@ def create_integration(
                 "commonfields": {"id": name, "version": -1},
                 "name": name,
                 "display": name,
-                "description": f"this is an integration {name}",
+                "description": description
+                if description is not None
+                else f"this is an integration {name}",
```

Comment:
```suggestion
                "description": description or f"this is an integration {name}",
```

Reply:
I did it like that originally, but the `black` formatter (which seems to be required to run in order to pass validations) changed it to look that way.
You can see all of its formatting changes on c5a12db2fc507e4ad33fafd0afa783f745dd66a4

Reply:
black only changes the structure (line breaking etc), not the content. I second @bziser's suggestion.

Reply:
Also @bziser, I added the `is not None` since I wanted an empty string description creation to be possible (in case we want to test such a case)

Reply:
@dorschw Nope, seems like black does care about line breaks, as it fails now.
https://app.circleci.com/pipelines/github/demisto/demisto-sdk/22074/workflows/f4a777ce-8094-4f31-ada4-6e04ffb3c3c5/jobs/170704

Following commit 60abcb1047338b8a9e0995a4becd4bfe3b131b6e

Reply:
that's just a test object, I wouldn't worry too much about it.